guide/getting-started/personal-finance.md

```diff
@@ -36,13 +36,13 @@ Illustration sources
 # Personal finance
 {:.no_toc}
 
-**If we consider bitcoin a new form of money, then we should consider how we interact with existing forms of money and how bitcoin compares. This can guide us towards behavioral patterns that users may already be familiar with.**
+**If we consider Bitcoin a new form of money, we should consider how we interact with existing forms of money and how Bitcoin compares. This can guide us towards behavioral patterns that users may already be familiar with.**
 
-Money is complex and can play many different roles in our lives. Buying a new shirt is a very different transaction than buying a house or setting aside savings for retirement, and taking a payment as a store owner is very different from a child receiving allowance from parents. On a global scale, the differences multiply across countries and cultures.
+Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.
 
-It still is possible to create simplified mental models that help us better understand user needs and behaviors. A single model might be oversimplified, but if we combine a few different lenses we can narrow in on a more complete picture that allows us to create more focused experiences, have better product differentiation, and make it easier for contributors to collaborate to make decisions.
```

Comment:
> A single model might be oversimplified, but if we combine a few different lenses we can narrow in on a more complete picture that allows us to create more focused experiences, have better product differentiation, and make it easier for contributors to collaborate to make decisions.

Why remove?

Reply:
Quite honestly, it sounded complicated, in the hindsight. We should tackle removals in a different or,

Reverted https://github.com/BitcoinDesign/Guide/pull/170/commits/8796246fd2cb09bb9065848f6f28145b3566637d